.eslintrc.js

```diff
@@ -14,9 +14,14 @@ module.exports = {
     project: true,
     tsconfigRootDir: __dirname
   },
-
+  plugins: [
+  ],
   rules: {
     eqeqeq: ['error', 'always'],
+    '@typescript-eslint/consistent-type-definitions': 'off',
+    '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/indent': 'off',
```

Comment:
honestly this one is actually a helpful one to keep - any reason you think we should not enforce indentation depths?

Reply:
Will keep